git_machete/git_operations.py

```diff
@@ -865,14 +864,14 @@ def get_latest_checkout_timestamps(self) -> Dict[str, int]:  # TODO (#110): defa
                     result[to_branch] = int(match.group(1))
         return result
 
-    def get_commit_information(self, information: str, commit: Optional[AnyRevision] = None) -> str:
-        if information not in GIT_FORMAT_PATTERNS:
+    def get_commit_information(self, commit: AnyRevision, information: GitFormatPatterns) -> str:
+        if information not in GitFormatPatterns:
```

Comment:
This check is probably redundant now right? I mean, it doesn't seem possible to pass a value of `information` that would NOT be `in GitFormatPatterns` without some heavy hacking (assuming mypy still passes), right?

Reply:
Well, you can still make a TYPO or type enum field that is not in the class.

e.g. 

`class A(Enum):
   FIELD_1 = 1

A.FEILD_1
`